lib/rules/lines-between-class-members.js

```diff
@@ -120,20 +93,26 @@ module.exports = {
                     const curFirst = sourceCode.getFirstToken(body[i]);
                     const curLast = sourceCode.getLastToken(body[i]);
                     const nextFirst = sourceCode.getFirstToken(body[i + 1]);
-                    const isPadded = isPaddingBetweenTokens(curLast, nextFirst);
                     const isMulti = !astUtils.isTokenOnSameLine(curFirst, curLast);
                     const skip = !isMulti && options[1].exceptAfterSingleLine;
-
+                    const beforePadding = findLastConsecutiveTokenAfter(curLast, nextFirst, 1);
+                    const afterPadding = findFirstConsecutiveTokenBefore(nextFirst, curLast, 1);
+                    const isPadded = afterPadding.loc.start.line - beforePadding.loc.start.line > 1;
```

Comment:
Feels more correct to use `beforePadding.loc.end.line` here? That would also probably require to check if `afterPadding` and `beforePadding` are the same token, in order to avoid mistake with a single multiline comment between class members.

It's probably impossible to design an example where this can make a difference with the current syntax, but I think it makes more sense and is easier to understand than comparing start lines.

With the future syntax (or present with `babel-eslint`), this would be an example:

```js
class A {
  x = `multiline
  template string`
  y = 1
}
```

(to avoid confusion - there is no need to add `babel-eslint` test).